connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java

```diff
@@ -221,28 +223,44 @@ public boolean isRunning() {
     }
 
     @SuppressWarnings("fallthrough")
-    private void pause() {
+    private void stop(boolean paused) {
```

Comment:
nit: keep pause(), add stop(), and have them both delegate to suspend(boolean)?

Reply:
I do like `suspend` better than `stop`, but IMO it's not worth the extra layer of indirection to add `stop` and `pause` methods that just delegate to that method with a single parameter changed; might make the code base harder to read for newcomers.